Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/WANDPowderReduction.py

```diff
@@ -130,119 +132,378 @@ def PyInit(self):
             MatrixWorkspaceProperty("OutputWorkspace", "", direction=Direction.Output),
             doc="Output Workspace",
         )
+        
+        self.declareProperty(
+            "MultipleOutput",
+            "",
+            StringListValidator(["Yes", "No"]),
+            doc="Specifies either single output workspace or output group workspace containing several workspaces",
+        )
+
 
     def PyExec(self):
-        data = self.getProperty("InputWorkspace").value  # [1~n]
```

Comment:
Convert all of the input data into a list of workspaces
```python
summing = self.getProperty("Sum").value  # [Yes or No]
data = self.getProperty("InputWorkspace").value  # [1~n]
bkg = self.getProperty("BackgroundWorkspace").value  # [1~n]
if len(data) == 1:
  data = [data]
elif len(data) > 1 and summing:
  temp = mtd[data[0]]
  for other in data[1:]:
    temp += mtd[other]
  data = [temp]
else:
  pass  # otherwise it is already a list
```